src/index.ts

```diff
@@ -1,16 +1,10 @@
-import BotDetector from './detector'
-import { BotDetectorInterface, InitOptions } from './types'
+import BotDetector from "./detector";
+import { BotDetectorInterface } from "./types";
 
-/**
- * Builds an instance of the BotDetector. It's recommended to call it as early as possible, ideally during application startup.
- *
- * @param {InitOptions} options Configuration options.
- * @returns {Promise<BotDetectorInterface>} A promise to the instance of the bot detector.
- */
-export async function load(options: InitOptions): Promise<BotDetectorInterface> {
-  const detector = new BotDetector(options)
-  await detector.collect()
-  return detector
+export async function load(): Promise<BotDetectorInterface> {
```

Comment:
@r-valitov said that he is going to use BotD as a dependency for Fingerprint Pro. It means that BotD must export its sources. Are you going to implement it in this PR or later?

I'd implement the connection with Fingerprint Pro before releasing v1.0.0 just in case. Such way you'll be able to change the BotD interfaces without worrying about the backward compatibility.